projects/fsastorefrontlib/src/cms-components/change-process/change-simulation/change-simulation.component.html

```diff
@@ -130,7 +130,7 @@
         class="primary-button btn-block"
         (click)="updateChangeRequest(changeRequest)"
       >
-        {{ 'common.submit' | cxTranslate }}
+        {{ 'fscommon.submit' | cxTranslate }}
```

Comment:
We should perhaps delete our submit key, since there is already Spartacus implementation which we can use? No need for duplication. As far as I can see we use our `'fscommon.submit'` only on `fnol-navigation.component`

Reply:
Spartacus doesn't have a German translation for `common.submit`.